src/FastJetTopTagger.cpp

```diff
@@ -286,6 +362,22 @@ void FastJetTopTagger::processEvent(LCEvent * evt){
   
 } //end processEvent
 
+double FastJetTopTagger::getECF(PseudoJet& jet, int whichECF, double beta, std::string energyCorrelator){
```

Comment:
pass the `energyCorrelator` as constant reference (`std::string const& energ...`), to avoid making copies of the object every time the function is called.

Reply:
done